### ISSUE TITLE
[SCREENREADER]: Colmery 107 - We need an error when users press Search School without entering a search term

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2381]
## Description

When users try to search for a school without entering a search term on the GI Bill Comparison Tool, no search is executed and it fails silently. Users would benefit from an error message so they know what their next action should be. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, I want to hear an error message read out when I press `Search Schools` without entering a term like "University of Mississippi" in the search field.

## WCAG or Vendor Guidance (optional)

* [Error Identification: Understanding SC 3.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-identified.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-10 at 4 37 06 PM](https://user-images.githubusercontent.com/934879/66608837-18249200-eb7d-11e9-9b8d-1d795cc4d9ab.png)

**Comment by tmcmurdo** [comment]
@peggygannon To address the issue mentioned above, the BAH team is considering the following error message:
“Please enter a city, school or employer name.”

Please let us know if you would like us to modify it in anyway.Thanks!

**Comment by peggygannon** [comment]
@tmcmurdo Yes, that inline error works. per our style, we should add a comma after school (in the inline error and in the label). 
 
Please enter a city, school, or employer name.

**Comment by tmcmurdo** [comment]
@hughes-dustin Seems like we'd want to add the extra comma to the label, as well. @dneel-bah would that be a different story?

**Comment by hughes-dustin** [comment]
@tmcmurdo Are oxford commas part of our standards?

**Comment by hughes-dustin** [comment]
Branch: bah-2381
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/10899

**Comment by hughes-dustin** [comment]
![Screen Shot 2019-10-15 at 10.38.19 AM.png](https://images.zenhubusercontent.com/5c98ddbd98ed1e00013adfec/fc9d9aa1-7f45-409c-9126-23edaf281398)

**Comment by delli-gatti-michael** [comment]
Local testing passes

![Screen Shot 2019-10-21 at 1.32.57 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/05027a0f-1761-4989-8e6d-06e15cd00dab)

**Comment by 1Copenut** [comment]
Passes testing in VoiceOver on Safari (MacOS Mojave) as well. LMK when it's in staging and I'll re-test it against NVDA and JAWS.

**Comment by delli-gatti-michael** [comment]
@1Copenut This is now in Staging and ready for re-testing

**Comment by 1Copenut** [comment]
@delli-gatti-michael I tested on Staging in JAWS and NVDA with preferred browsers. All fired and announced the error as expected. Moving to close.
